crashlogger.js

```diff
@@ -45,18 +61,22 @@ exports = module.exports = function (err, description, data) {
 			console.error(`Could not start nodemailer - try \`npm install\` if you want to use it`);
 		}
 		if (transport) {
-			transport.sendMail({
-				from: Config.crashguardemail.from,
-				to: Config.crashguardemail.to,
-				subject: Config.crashguardemail.subject,
-				text: `${description} crashed ${exports.hadException ? "again " : ""}with this stack trace:\n${stack}`,
-			}, err => {
-				if (err) console.error(`Error sending email: ${err}`);
-			});
+			transport.sendMail(
+				{
+					from: Config.crashguardemail.from,
+					to: Config.crashguardemail.to,
+					subject: Config.crashguardemail.subject,
+					text: `${description} crashed ${hadException ? "again " : ""}with this stack trace:\n${stack}`,
+				},
+				/** @param {Error?} err */
```

Comment:
Really, the param types should be inferred from the `createServer` signature. Specifying params here is a last resort that shouldn't usually be done.